modules/artifact/src/test/java/org/mule/runtime/module/artifact/classloader/CompositeClassLoaderTestCase.java

```diff
@@ -96,9 +106,12 @@ public void usesParentFirstThenChildLookup() throws ClassNotFoundException
         assertThat(aClass, equalTo(CLASS_FROM_CLASSLOADER1));
     }
 
-    @Test(expected = ClassNotFoundException.class)
+    @Test
     public void  usesParentFirstAndChildLookupAndFails() throws ClassNotFoundException
     {
+        expected.expect(CompositeClassNotFoundException.class);
+        expected.expectMessage(startsWith("Cannot load class '" + CLASS_NAME + "': ["));
```

Comment:
Should not assert that there are multiple exceptions in the composite exception?